File: Python/fibo.py
print("Enter a number:")
n = int(input())
f1, f2, sum = 0, 1, 0
if (n == 0):
    print(0)
else:
    print("Fibonacci Series:", f1, f2, end=' ')
    for i in range(2, n):
        sum = f1+f2
        print(sum, end=' ')
        f1 = f2
        f2 = sum

File: Java/Fibo.java
import java.util.*;

class Fibo
{
    public static void main(String args[])
    {
        Scanner s=new Scanner(System.in);
        System.out.println("Enter Number:");
        int num=s.nextInt();
        int f1=0,f2=1,sum=0;
        System.out.print("Fibonacci Series:"+f1+" "+f2);
        for(int i=2;i<num;i++)
        {
            sum=f1+f2;
            System.out.print(" "+sum);
            f1=f2;
            f2=sum;
        }
        s.close();
    }
}